docs/reference/early-access.md

```diff
@@ -0,0 +1,56 @@
+---
+id: early-access
+title: Early access
+sidebar_label: Early access
+description: "Use early access releases to quickly learn about upcoming features and components."
+---
+
+Use early access releases to quickly learn about upcoming features and components.
+
+## Types of early access releases
+
+Selected Camunda features and components are released upfront as alpha or beta versions, e.g., to provide users the opportunity to test a feature and participate in the development by giving feedback prior to market launch (general availability, GA).
+
+Early access releases are made available outside of the [release policy](/reference/release-policy.md).
+
+### Alpha
+
+Alpha features and components are introduced to gather early feedback from users. These features and components are still under heavy development, and might change. These releases are intended only for non-production usages in trying out recent, potentially yet unfinished new features. Characteristics are:
+
+- Not for production use
+- Unstable, potentially causing performance issues
+- APIs, dependencies and configuration are likely to change
+- Not necessarily feature-complete
+- May not be documented
+- No update support guaranteed
+- Not covered by service level agreements (SLAs), and no support obligations
+
+### Beta
+
+The scope of beta features and components is close to that of the targeted GA release. A beta release may still be missing some features. Characteristics are:
```

Comment:
It's important to highlight the differences. Here, the only practical difference is that update support is guaranteed.
"The scope of beta features and components is close enough to the targeted GA release that we are willing to provide update support, but we're still making the final changes. Therefore, it may still be missing some features. Characteristics are:"

Reply:
I like that one